app/views/message_threads/index.html.erb

```diff
@@ -3,14 +3,23 @@
     <meta name="turbo-visit-control" content="reload">
   <% end %>
 <% end %>
-
 <%= render MessageThreadsTableComponent.new(filter: @filter, filter_subscription: @filter_subscription) do |component| %>
-  <% component.with_message_thread do %>
-    <%= render MessageThreadsTableRowComponent.with_collection(@message_threads) %>
-  <% end %>
-  <% if @next_cursor %>
-    <% component.with_next_page_area do %>
-      <%= render Turbo::NextPageAreaComponent.new(id: @next_cursor, url: scroll_message_threads_url(@next_page_params)) %>
+  <% if @message_threads.any? %>
+    <% component.with_message_thread do %>
+      <%= render MessageThreadsTableRowComponent.with_collection(@message_threads) %>
+    <% end %>
+    <% if @next_cursor %>
+      <% component.with_next_page_area do %>
+        <%= render Turbo::NextPageAreaComponent.new(id: @next_cursor, url: scroll_message_threads_url(@next_page_params)) %>
+      <% end %>
+    <% end %>
+  <% else %>
+    <% component.with_blank_results_area do %>
+      <% if params[:referrer] == "menu" %>
+        <%= render Common::BlankResultsComponent.new(:all_done) %>
```

Comment:
Toto neviem ci je uplne spravna logika, ak "idem z menu", tak mozem mat aj filter "Vybavene" a ked je prazdny tak nie je prave naopak NIC vybavene. Zaroven to posuvanie referera sa mi nepaci.

Spravme teda len tu verziu, ze ziadne spravy.